docs/markdown/star_formation.md

```diff
@@ -5,7 +5,7 @@ The star formation recipe is implemented via the Stochastic Stellar Population s
 
 Adding a star particle to every Jeans violating cell can lead to a large number of small mass stars which can be cumbersome. To alleviate this issue, we control the star formation rate through $\epsilon_{\rm ff}$ and $\epsilon_{*}$. $\epsilon_{\rm ff}$ is the efficiency per freefall time given by $\frac{\dot{M}_{*} \cdot t_{\rm ff}}{M_{\rm cell} \cdot dt}$, while $\epsilon_{*}$ is the fraction of cell mass that gets converted into stars. 
 
-We emulate the stochasticity of star formation in the following manner. We associating a probability P to the process of spawning a star in the cell. A choice for P $=\epsilon_{\rm ff} \cdot dt/\epsilon_{*}$, which is ratio of the expected from the star formation rate in the cell over the timestep dt ($\epsilon_{\rm ff} dt/t_{\rm ff}$) to the fraction of mass available for star formation ($\epsilon_{\rm ff} M$). It should be noted that this is the probability for $\epsilon_{\rm ff}$ averaged over the timestep rather than for the instantaneous star formation rate, which should be integrated over the timestep dt. 
+We emulate the stochasticity of star formation in the following manner. We associating a probability P to the process of spawning a star in the cell. A choice for P $=\epsilon_{\rm ff} \cdot dt/\epsilon_{*}/t_{\rm ff}$, which is ratio of the expected from the star formation rate in the cell over the timestep dt ($\epsilon_{\rm ff} dt/t_{\rm ff}$) to the fraction of mass available for star formation ($\epsilon_{\rm ff} M$). It should be noted that this is the probability for $\epsilon_{\rm ff}$ averaged over the timestep rather than for the instantaneous star formation rate, which should be integrated over the timestep dt. 
```

Comment:
Typo here: "We associating" --> "We associate". Also, I think the explanation in the sentence that starts "A choice for P" is a bit hard to follow. I suggest rewriting as follows:

To set $P$, we note that the expectation value of the stellar mass formed is $P$ multiplied by the mass of the star that will be inserted, $\epsilon_* M_\mathrm{cell}$, so $\langle M_*\rangle = P \epsilon_* M_\mathrm{cell}$. This must match the target determined by $\epsilon_\mathrm{ff}$, which is $\langle M_*\rangle = \epsilon_\mathrm{ff} M_\mathrm{cell} (dt / t_\mathrm{ff}$; this in turn requires that $P = (\epsilon_\mathrm{ff}/\epsilon_*) (dt / t_\mathrm{ff})$.